src/helpers/config.ts

```diff
@@ -1,4 +1,3 @@
-import fs from "node:fs";
```

Comment:
Details on the difference between "node:fs" and "fs" are here: https://nodejs.org/api/modules.html#core-modules
TL;DR; the `node:` prefix ignore the require cache.  In our case we don't want to do that because the SDK consumer may want to be able to leverage the require cache.